### DISCUSSION TITLE
Transferencia de datos de Base de datos

### DISCUSSION THREAD

**Discussion by BrayanGonzalez24** [dbeaver/dbeaver#19442] [Q&A] [answered]
¿Quisiera saber cual es el limite para transferir datos de una tabla en una base de datos a otra tabla de otra base de datos?

Automatic translation:
I would like to know what is the limit to transfer data from a table in a database to another table in another database?

**Comment by LonwoLonwo** [comment]
Hello @BrayanGonzalez24 
DBeaver does not have such limits by default. It has some java limits that can be changed with https://github.com/dbeaver/dbeaver/wiki/Command-Line#vm-arguments
Other limits depend on Databases and their drivers.

**Comment by BrayanGonzalez24** [comment]
Osea puedo transferir 100 millones de datos de una tabla a otra tabla de diferente BD?

**Comment by BrayanGonzalez24** [comment]
Tengo las BD en informix

**Comment by ShadelessFox** [comment, marked as answer]
Yes, you can.

Please let us know if you have any questions left.
